js/components/multi-audio/multiAudioPopover.js

```diff
@@ -38,11 +41,17 @@ var MultiAudioPopover = React.createClass({
             closeAction={this.handleClose} />
         </li>
         {multiAudio.map(function(audio){
+          var color = "white";
+          if (audio.enabled) {
+            color = "blue";
+          }
+
           return (
             <li>
               <span
                 className="oo-multiaudio-element"
                 onClick={this.handleMultiAudioClick.bind(this, audio.id)}
+                style={{color:color}}
```

Comment:
I know that this is temporary, but I think this should be a class instead of a style. The customer might want to customize the appearance of selected items. I think we should also add this scenario to the unit tests of the multi audio popover component.

Reply:
Now, user doesn’t see a functionality of multiaudio. It was added for checking and testing by the developers. In tasks “https://jira.corp.ooyala.com/browse/PLAYER-542” and “https://jira.corp.ooyala.com/browse/PLAYER-522" the interface will be completely redesigned and css classes will be added. Now, it makes no sense to refactor and add unit tests.